src/components/cluster/detail/cluster_apps.js

```diff
@@ -168,20 +168,28 @@ class ClusterApps extends React.Component {
     return (
       <React.Fragment>
         {this.props.showInstalledAppsBlock && (
-          <div id='installed-apps-section'>
+          <div data-testid='installed-apps-section' id='installed-apps-section'>
```

Comment:
Is there a way around having this `data-testid?` at first glance, not really liking that we have id and data-testid.